task_29/Anton.Panciuc/src/main/java/com/hillel/homework/Cesar.java

```diff
@@ -0,0 +1,42 @@
+package com.hillel.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cesar {
+  List<Character> alphabet = new ArrayList<>();
```

Comment:
Make it as constant and specify the elements directly without a constructor.
You can use a plain String here instead of a List with all letters.